src/IECoreMaya/SceneShapeInterface.cpp

```diff
@@ -1094,6 +1094,18 @@ MPxSurfaceShape::MatchResult SceneShapeInterface::matchComponent( const MSelecti
 	return MPxSurfaceShape::matchComponent( item, spec, list );
 }
 
+MSelectionMask SceneShapeInterface::getShapeSelectionMask() const
```

Comment:
Am I to understand this is working again?

Reply:
Nope... But these bits are definitely gonna be needed, I think.